### PR TITLE
Remove PHP versions without support nor security fix support

### PR DESCRIPTION
**Pull request by specialtactics**
I just noticed there are a lot of really old PHP versions in the testing, I think they are prone to cause issues and incompatibilities, and they shouldn't really be encouraged to be used as they haven't been supported for a long time.

For reference:
https://www.php.net/supported-versions.php

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,4 @@
 composer.lock
 build
 coverage.xml
+.idea
```
Comment (mfn) [inline .gitignore:5]: This shouldn't be in here. Put your IDEs meta folder in your global gitignore, please.

Comment (specialtactics) [reply .gitignore:5]: What's the issue with it being here, @mfn?

Comment (mfn) [reply .gitignore:5]: It doesn't make sense to have an ever expanding list of all the editors/IDEs in each code repository people work on. Next one wil add `.vscode`, then `.sublime` or `.eclipse` etc.

These should be in each developers global ignore.

Comment (specialtactics) [reply .gitignore:5]: What's wrong with that? There aren't really that many IDEs, I think it's fairly standard to have that in .gitignore.

**Comment by tymondesigns** [comment]
Going to cleanup the php requirements soon, will also be moving from travis to github actions

**Comment by tymondesigns** [comment]
closing as moved to github actions

**Pull request closed without merging**